kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/appliers/tasks/VariantReportsSet.kt

```diff
@@ -117,6 +136,9 @@ internal class VariantReportsSet(
             onlyIf {
                 fileWithMessage.asFile.get().exists()
             }
+
+            // always execute
+            outputs.upToDateWhen { false }
```

Comment:
Why do we need to always execute this task? So it reports the same warning over and over?

Reply:
So that the task behaves consistently.

If you call task twice in a row
```
% ./gradlew :koverVerify 
> Task :koverVerify 
Kover Verification Error
Rule violated: lines covered percentage is 10, but expected minimum is 79.0000
BUILD SUCCESSFUL in 8s

// since some changes

% ./gradlew :koverVerify 
BUILD SUCCESSFUL in 8s
```
There may be a mistaken impression that the error has gone away. However, the task was in `UP-TO-DATE` status because the source code files on which it depends were not changed.